backend/player/human.js

```diff
@@ -145,29 +141,35 @@ module.exports = class Human extends Player {
     this.selected = {
       picks: [],
       burns: []
-    }
+    };
 
     this.updateDraftStats(this.draftLog.pack, this.pool);
 
     this.emit("pass", pack);
   }
   handleTimeout() {
```

Comment:
This was they other area that needed tweaking to get auto - top up of missing picks working